Entities/Scene.cpp

```diff
@@ -3017,17 +3053,17 @@ void Scene::Update()
 		}
 	}
 
-    // Do full update every two minutes
-    if (m_FullPathUpdateTimer.IsPastSimMS(120000))
+    // Do a partial update every 10 seconds
+    if (m_PartialPathUpdateTimer.IsPastRealMS(10000)) 
     {
-        m_pPathFinder->RecalculateAllCosts();
-        m_FullPathUpdateTimer.Reset();
-        m_PathfindingUpdated = true;
+        UpdatePathFinding();
     }
+}
 
-    // Do partial update every 10 seconds
-    if (m_PartialPathUpdateTimer.IsPastRealMS(10000))
-        UpdatePathFinding();
+PathFinder* Scene::GetPathfinder(Activity::Teams team) 
+{
+    int i = static_cast<int>(team) + 1; // + 1 because of our shared NoTeam pathfinder (index -1 becomes 0)
```

Comment:
I think the comment is valuable here, but defining i on a separate line seems unnecessary. I'd say tweak things a bit, like so:
```cpp
Pathfinder * Scene::GetPathFinder(Activity::Teams team)
{
    // Note that we use team + 1 here, because of our shared NoTeam pathfinder. Team -1 becomes index 0 in the array.
    return m_pPathFinders[static_cast<int>(team) + 1];
}
```
Also, don't forget to move the `*`!